src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,5 @@
+package nextstep.fp;
+
+public interface Conditional {
```

Comment:
```suggestion
@FunctionalInterface
public interface Conditional {
```

함수형 인터페이스임을 애너테이션으로 명시하면 더 좋겠네요!
애너테이션이 명시된 인터페이스에 추가 메소드를 정의하면 IDE 자체에서도 경고가 뜹니다!